cmd/ocitools/validate.go

```diff
@@ -61,6 +62,8 @@ var (
 		"CAP_KILL",
 		"CAP_AUDIT_WRITE",
 	}
+
+	configSchemaTemplate = "https://raw.githubusercontent.com/opencontainers/runtime-spec/v%s/schema/schema.json"
```

Comment:
On Thu, Aug 18, 2016 at 11:41:17AM -0700, Mrunal Patel wrote:
> > @@ -61,6 +62,8 @@ var (
> >  		"CAP_KILL",
> >  		"CAP_AUDIT_WRITE",
> >  	}
> > +
> > +	configSchemaTemplate = "https://raw.githubusercontent.com/opencontainers/runtime-spec/v%s/schema/schema.json"
> 
> We should vendor this file rather than downloading it. 

If we vendor (and I don't see a huge need for that), I'd rather vendor
with the option to download (opencontainers/image-spec#150) ;).  The
drawback to *only* vendoring is that if we don't vendor (and
runtime-spec is consistent about the naming pattern) we can
automatically validate against *future* schema versions that weren't
available at compile time.

I am a fan of not requiring network activity, but I'd rather support
that with a ‘--schema $URL’ option (something like
opencontainers/runtime-spec#490) so users could run:

  $ oci validate --schema file:///path/to/config-schema.json …